lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -3162,6 +3162,7 @@ Name | Description
 ``$__user_id__`` | Email's numeric ID (id column of ``galaxy_user`` table in the database)
 ``$__user_email__`` | User's email address
 ``$__app__`` | The ``galaxy.app.UniverseApplication`` instance, gives access to all other configuration file variables (e.g. $__app__.config.output_size_limit). Should be used as a last resort, may go away in future releases.
+``$__target_datatype__`` | Only available in converters. Contains the target datatype of the conversion
```

Comment:
```suggestion
``$__target_datatype__`` | Only available in converter tools when run internally by Galaxy. Contains the target datatype of the conversion
```

Maybe also add a small paragraph below suggesting to also add a `__target_datatype__` input param  to be able to run the converter tool manually, and a link to lib/galaxy/datatypes/converters/biom.xml as an example.